pkg/cri/resource-manager/policy/builtin/topology-aware/mocks_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/intel/goresctrl/pkg/sst"
 	idset "github.com/intel/goresctrl/pkg/utils"
 	v1 "k8s.io/api/core/v1"
-	cri "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+	cri "k8s.io/cri-api/pkg/apis/runtime/v1"
```

Comment:
Now that you renamed (almost) all the cri imports would we want to use `criv1` everywhere, for consistency? Again, I'll leave up to you. I think we merge PR also as is.

Reply:
Definitely. I just haven't realized that I missed some. Bad grep pattern... And ditto for the other remaining inconsistencies in `cache`.